releng/release-tool/release-tool.go

```diff
@@ -76,23 +78,26 @@ func _gitCommand(arg ...string) (string, error) {
 }
 
 func (r *releaseData) generateReleaseNotes() error {
-	additionalResources := fmt.Sprintf(`Additional Resources
---------------------
-- Mailing list: <https://groups.google.com/forum/#!forum/kubevirt-dev>
-- Slack: <https://kubernetes.slack.com/messages/virtualization>
-- An easy to use demo: <https://github.com/%s/demo>
-- [How to contribute][contributing]
-- [License][license]
-
-
-[contributing]: https://github.com/%s/%s/blob/main/CONTRIBUTING.md
-[license]: https://github.com/%s/%s/blob/main/LICENSE
----
+	additionalResources := fmt.Sprintf(`
+## Additional Resources
+
+- Mailing list: https://groups.google.com/forum/#!forum/kubevirt-dev
+- Slack: https://kubernetes.slack.com/messages/virtualization
+- An easy to use demo: https://github.com/%s/demo
+- How to contribute: https://github.com/%s/%s/blob/main/CONTRIBUTING.md
+- License: https://github.com/%s/%s/blob/main/LICENSE
 `, r.org, r.org, r.repo, r.org, r.repo)
 
 	tagUrl := fmt.Sprintf("https://github.com/%s/%s/releases/tag/%s", r.org, r.repo, r.tag)
 
-	r.releaseNotesFile = fmt.Sprintf("%s/%s-release-notes.txt", r.repoDir, r.tag)
+	releaseNotesDir := fmt.Sprintf("%s/CHANGELOG", r.repoDir)
```

Comment:
What about using `path.Join` instead of `fmt.Sprintf`?

Reply:
I've followed what was already there but your suggestion makes sense and, since I'm already touching the code, I might as well improve upon it :)

Reply:
Done.